deploy/charts/cert-manager/templates/BUILD.bazel

```diff
@@ -17,7 +17,7 @@ modify_file(
     name = "crds.legacy",
     src = "//deploy/crds:templates.legacy",
     out = "crds.legacy.yaml",
-    prefix = """{{- if (semverCompare "<1.15-0" .Capabilities.KubeVersion.GitVersion) }}
+    prefix = """{{- if (semverCompare "<1.16-0" .Capabilities.KubeVersion.GitVersion) }}
```

Comment:
Should we add a more descriptive release note, explaining the new  installation yaml for 1.15 users....or will that be done in a separate upgrading section of the release notes?

Reply:
This is a very big release note :) will prepare that PR this week so we have it in time for beta.0